src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -465,7 +481,7 @@ private Task CloseAsync(ConnectionContext connection)
             }
         }
 
-        private async Task SendHubMessage(HubInvocationMessage hubMessage, CancellationToken cancellationToken = default)
+        internal async Task SendHubMessage(HubMessage hubMessage, CancellationToken cancellationToken = default)
```

Comment:
This should be able to stay `private` I think.

Reply:
Changed it intending to use it in the unit tests, because it's where ```PingInterval``` is actually reset. Ending up using the less generic ```SendAsync``` to activate the reset, but forgot to switch it back.

Reply:
Don't forget to swap this back to `private` ;)